endpoints/openrtb2/sample-requests/valid-whole/supplementary/req-ext-bidder-params-backward-compatible-merge.json

```diff
@@ -24,16 +24,17 @@
           ]
         },
         "ext": {
-          "appnexus": {
-            "placementId": 111111
-          },
-          "pubmatic": {
-            "wrapper": {
-              "profile": 1234
-            }
-          },
           "prebid": {
-            "bidder": {}
+            "bidder": {
+              "appnexus": {
+                "placementId": 111111
+              },
+              "pubmatic": {
+                "wrapper": {
+                  "profile": 1234
+                }
+              }
+            }
```

Comment:
Why is this test affected?

Reply:
This test was affected because it had an empty `imp.ext.prebid.bidder` object defined in the request, and all the bidders were specified outside of it.  According to the new logic if there is an `imp.ext.prebid.bidder` object (even if it is empty one) - we must consider that the publisher has switched to the new way of specifying bidders and ignore the bidders in `imp.ext` - treat them as arbitrary fields and pass along as is.   Thus we had 3 options: 
1. modify the test fixture and remove an empty `prebid.bidder: {}`
2. modify the test fixture and promote the bidders from `imp.ext` into `imp.ext.prebid.bidder`
3. Modify the condition of promotion in request_validator.go:  instead of promoting only when `prebid.Bidder==nil` here:
```
if prebid.Bidder == nil {
    prebid.Bidder = make(map[string]json.RawMessage)
    bidderPromote = true
}
```
also promote in case it is empty: `if prebid.Bidder == nil || len(prebid.Bidder) == 0`
but this would disrupt the other tests logic. 

So we took option 2.  I think route 1 is less invasive, so I am adding a lighter modification to this test fixture - where it will only remove an empty `prebid.bidder: {} ` object.